backend/code/archipelag/event/migrations/0001_initial.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
```

Comment:
I assume that you know what you are doing. Changing old migration is almost always a bad idea. And I know that the application is not ready to be deployed, so this is not very bad, but please be aware.